BinanceExchange.API/Trade/BinTrade.cs

```diff
@@ -0,0 +1,191 @@
+﻿using System;
+using System.Threading.Tasks;
+using BinanceExchange.API.Client;
+using BinanceExchange.API.Enums;
+using BinanceExchange.API.Models.Request;
+
+//"dynamic" requires a reference to Microsoft.CSharp
+
+namespace BinanceExchange.API.Client.Trade
+{
+    public class MarginTrade
+    {
+        /// <summary>
+        /// Send isolated account buy command
+        /// </summary>
+        /// <param name="client"></param>
+        /// <param name="symbol">the trading symbol (BTCUSDT...)</param>
+        /// <param name="asset">for which the buy power is queried</param>
+        /// <returns></returns>
+        public static async Task<decimal> MaxBuyPowerInAsset(BinanceClient client, string symbol, string asset)
+        {
+            dynamic accountInfo = await client.QueryIsolatedMarginAccountInfo(new IsolatedMarginAccountInfoRequest()
```

Comment:
Also `dynamic` here. Take the response JSON and then create an associated C# class we can use 👍